mocha.opts

```diff
@@ -1,2 +1,2 @@
 --compilers js:babel/register
-@(components|widgets|lib|decorators)/**/*-test.js
```

Comment:
why would it not be mandatory? It was mapped on what most of the react community does (https://github.com/rackt/react-router/tree/master/modules/__tests__)